### PR TITLE
New job for running "autobuild/transifex-push-template.sh"

### PR DESCRIPTION
**Pull request by AnishGG**
Issue: #1150 
@ice0 
I have pushed a new job and removed all other jobs for the sake of fast testing. But the job is failing.
Is this expected behaviour?

### CODE REVIEW

**Comment by ice0** [comment]
@AnishGG Yes, there is an error in the .po file. So the script should fail.

**Comment by AnishGG** [comment]
Alright!
I will add all other jobs and ask for a review then.

File: .travis.yml [hunk 1]
```diff
@@ -90,6 +90,12 @@ matrix:
     script: ./travis.sh
     after_script: ccache -s # show ccache stats
 
+  - os: linux
+    name: "Transifex push template Ubuntu 16.04 Xenial"
+    sudo: required
+    dist: xenial
+    script: bash autobuild/transifex-push-translations.sh
```
Comment (ice0) [inline .travis.yml]: Also i suggest to change shell inside `transifex-push-translations.sh` from `/bin/sh` to `/bin/bash`, as `pushd/popd` cannot be executed by `sh`

**Comment by AnishGG** [comment]
> Please do not forget to add condition:
> job must run only on `translations` branch.
> 
> https://docs.travis-ci.com/user/conditional-builds-stages-jobs/

Oh, I see.
According to my knowledge, if we push a commit to a specific branch(say 'X'), it should trigger the branch X's `.travis.yml`, right?
Please correct me if I am wrong.

**Comment by ice0** [comment]
> According to my knowledge, if we push a commit to a specific branch(say 'X'), it should trigger the branch X's `.travis.yml`, right?
> Please correct me if I am wrong.

Yes.

**Comment by AnishGG** [comment]
> > According to my knowledge, if we push a commit to a specific branch(say 'X'), it should trigger the branch X's `.travis.yml`, right?
> > Please correct me if I am wrong.
> 
> Yes, it triggers `.travis.yml` from that branch.

Then we don't need an `if` condition, right? Because we will be editing only that branch's `.travis.yml`.

**Comment by ice0** [comment]
> Then we don't need an `if` condition, right? Because we will be editing only that branch's `.travis.yml`.

The main idea was to update `translations` branch from `master` branch every time when we need to update translations. If we do different versions of `.travis.yml` when we need to resolve merge conflicts every time when we do updates to this branch.

**Comment by AnishGG** [comment]
> The main idea was to update `translations` branch from `master` branch every time when we need to update translations. If we do different versions of `.travis.yml` when we need to resolve merge conflicts every time when we do updates to this branch.

Oh!
Got it now. Then I will introduce the `if` condition!
One more doubt: should I make the P.R. to `master` branch or `translations` branch then?

**Comment by ice0** [comment]
> Oh!
> Got it now. Then I will introduce the `if` condition!
> One more doubt: should I make the P.R. to `master` branch or `translations` branch then?

To `master` branch :)

**Comment by AnishGG** [comment]
> > Oh!
> > Got it now. Then I will introduce the `if` condition!
> > One more doubt: should I make the P.R. to `master` branch or `translations` branch then?
> 
> To `master` branch :)

Alright!
I might have to open a new PR then. Will close this if required

**Comment by ice0** [comment]
You can change branch by editing this PR.

**Comment by AnishGG** [comment]
Okay, will try that!

**Pull request closed without merging**